doc/ref_impl/cntt-ri/chapters/chapter03.md

```diff
@@ -4,16 +4,56 @@
 <p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
-* [3.1 Introduction](#2.1)
-* [3.2 Heading](#2.2)
-* [3.3 Heading](#2.3)
+* [3.1 Introduction](#3.1)
+* [3.2 VNF profile](#3.2)
+* [3.3 NFVI SW profile](#3.3)
+* [3.4 NFVI HW profile](#3.4)
 
 <a name="3.1"></a>
 ## 3.1 Introduction
 
+Three types of profiles are mentioned in CNTT Reference Model: VNF profile is used to describe every workload running on top of NFVI. NFVI SW profile is used to describe the list of features provided by the hypervisor and host OS. Finally the NFVI HW profile is used to depict the details characteristics of every actual servers on the bottom.
+
+The CNTT Reference Model will be referenced as **RM1** to avoid long and duplicated reference titles.
+
 <a name="3.2"></a>
-## 3.2 Heading
+## 3.2 VNF profile
+
+Any virtual network functions and/or cloud-native network functions must choose to run on one of the pre-defined of entries in NFVI Infrastructure Profiles Catalogue. As states in [RM1: 4 Infrastructure Capabilities, Measurements and Catalogue: 4.2 Catalogue](https://cntt-n.github.io/CNTT/doc/ref_model/chapters/chapter04.html#4.2), the entry uses the following naming convention.
```

Comment:
For adding hyperlinks, use **relative md file linking** instead of linking them to hosted github.io.
For example, instead of using (https://cntt-n.github.io/CNTT/doc/ref_model/chapters/chapter04.html#4.2), 
(../../../ref_model/chapters/chapter04.md#4.2)

This will be automatically updated to github.io link when it will be hosted.

Fix this here and for other usages.

Reply:
Thanks a lot. I will update accordingly.

Also I have a question, I've installed Jekyll on my laptop. Is it possible to preview what the website looks like locally before I submit ?

I go to the root of our project and launch `jekyll serve` but I cannot get the website through http://127.0.0.1:4000. I can only view all these files from web browser.

Any more configuration I should apply for local preview.

Reply:
Default settings gives issues on local. It need some extra configurations. Below steps worked for me. You can try them if it works for you.
1. Install bundler. `gem install bundler`
2. Create a Gemfile in CNTT root directory with below content:
```
gem "github-pages", group: :jekyll_plugins

```
3. Serve the jekyll site. `bundle exec jekyll serve`

Site should be available with similar view as `https://cntt-n.github.io/CNTT/` at http://127.0.0.1:4000/.